app/src/main/java/org/simple/clinic/appconfig/Country.kt

```diff
@@ -21,4 +21,11 @@ data class Country(
 
     @Json(name = "isd_code")
     val isdCode: String
-) : Parcelable
+) : Parcelable {
+
+    companion object {
+        const val INDIA = "IN"
+        const val BANGLADESH = "BD"
+        const val ETHIOPIA = "ET"
```

Comment:
Are we reusing these constants elsewhere?
If not, then we can inline them instead of creating global values.

Reply:
I have seen Ragunath's PR too and this will be reused in other places too. Also, I find it nicer to have literal constants stored in one place than freely mentioning them all over our code?

Reply:
It make sense to do this once we need to reuse it not before. Let's not code ahead.
We can keep this as is for now because this should change once we come up with a way to set all country-specific configuration in one place.

Reply:
I have created a pivotol story to track this: https://www.pivotaltracker.com/story/show/170045234